File: app/src/main/java/in/woloo/www/utils/IOnCallWSCallBack.java
package in.woloo.www.utils;

import com.android.volley.VolleyError;

import org.json.JSONObject;

import in.woloo.www.more.models.FileUploadResponse;

public interface IOnCallWSCallBack {
    public void onSuccessResponse(FileUploadResponse fileUploadResponse);
    public void onFailure(VolleyError volleyError);
}

File: app/src/main/java/in/woloo/www/utils/IOnCallWSCallBack.kt
package `in`.woloo.www.utils

import com.android.volley.VolleyError
import `in`.woloo.www.more.models.FileUploadResponse

interface IOnCallWSCallBack {
    fun onSuccessResponse(fileUploadResponse: FileUploadResponse?)
    fun onFailure(volleyError: VolleyError?)
}
